### PR TITLE
Fix bug UpdateComment, Fix styling on Comment

### LINKED ISSUE #1718: 🐛 [Bug] Editing comments is not reactive again
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-25T09:12:31Z" title="Wednesday, September 25th 2019, 11:12:31 am +02:00">Sep 25, 2019</time>_
_Closed <time datetime="2019-09-26T21:14:35Z" title="Thursday, September 26th 2019, 11:14:35 pm +02:00">Sep 26, 2019</time>_
---

## :bug: Bugreport

It seems that we have broken the reactivity of our `UpdateComment` functionality again
At the moment, we are just adding the updated `Comment` to `this.post.comments`, which updates, but does not change the apollo cache and therefore is not updated on the page until a hard refresh.
This is the reason we programmed it to update the apollo cache before the refactor.
Of course, it seemed to have been working when the refactor was merged in, so maybe something else broke it??
On `nitro-staging` after comment was updated
Screenshot from 2019-09-25 11-02-55
Also, on staging, after hard refresh
Screenshot from 2019-09-25 11-03-07
In dev env., console logging `this.post.comments`
Screenshot from 2019-09-25 11-05-36

### Steps to reproduce the behavior
1. go to the site, reproducible on `alpha`, `nitro-staging`, and `localhost`
2. type a comment, if you don't have one to edit
3. edit a comment
4. observe that nothing changed
5. refresh the page
6. observe that the comment was successfully updated and viewable

### Expected behavior

we should have it updated for the user without the need of a hard refresh

### PR DESCRIPTION
**Pull request by Tirokk**
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-25T11:52:02Z" title="Wednesday, September 25th 2019, 1:52:02 pm +02:00">Sep 25, 2019</time>_
_Merged <time datetime="2019-09-26T21:14:35Z" title="Thursday, September 26th 2019, 11:14:35 pm +02:00">Sep 26, 2019</time>_
---

- There was a bug that updates to comments were not being displayed
because of a v-show... https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show
Basically, what it says that is important for this bug is
"v-if...ensures that event listeners and child components inside the
conditional block are properly destroyed and re-created during toggles"

- Since we are using the editor to display comments in ContentViewer, we
need to set the min-height to 0px

- Co-authored-by: Alina Beck <alina.beck@mail.com>
- Co-authored-by: Robert Schäfer <git@roschaefer.de>

### Issues

- fixes #1718

### CODE REVIEW

**Review by Mogge — APPROVED** [review]
Coolio!
cool
